packages/manager/modules/emailpro/src/emailpro.module.js

```diff
@@ -31,6 +31,14 @@ angular
   .constant('EMAILPRO_CONFIG_URL', EMAILPRO_CONFIG_URL)
   .constant('EMAILPRO_CONFIG', EMAILPRO_CONFIG)
   .config(routing)
+  .provider(
```

Comment:
Why a `provider` instead of a `constant` ?

Reply:
Because you can't declare constant depending on dynamic values